spec/rubocop/cop/rspec/preferred_shared_example_methods_spec.rb

```diff
@@ -0,0 +1,97 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::RSpec::PreferredSharedExampleMethods, :config do
+  shared_examples 'preferred method' do |config_key, methods|
```

Comment:
Nitpick: add  ‘preferred’ and ‘**others’ (or pass those as kwargs to indicate the intent when calling, too)